search-app/src/cmr/search/site/routes.clj

```diff
@@ -23,6 +23,12 @@
         (GET "/"
              {context :request-context}
              (pages/home context))
+        (GET "/sitemap.xml"
```

Comment:
We should pay attention to how long it takes to request each of these routes. For example for a provider with more than a thousand collections, how long does it take to generate this XML?

I'm thinking if the time is excessive we might want some kind of background job that periodically generates the XML behind the scenes and we just serve it up from cache.